algo/src/test/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactoryTest.java

```diff
@@ -21,12 +25,12 @@
 public class ActionableEvidenceFactoryTest {
 
     private static final TreatmentApproachCurator curator = TreatmentApproachTestFactory.createTestCurator();
-
+    private static final CkbEvidenceBlacklistModel blacklistEvidence = CkbBlacklistEvidenceTest.createCkbBlacklistEvidenceEmpty();
```

Comment:
Creating an empty model should be done from a helper (test) factory class rather than another test class

Reply:
done